apps/eth/tasks.py

```diff
@@ -2,27 +2,30 @@
 
 from celery import shared_task
 from celery_once import QueueOnce
-from rest_framework.exceptions import APIException
+from django.conf import settings
 from influxdb_metrics.loader import log_metric
-
+from rest_framework.exceptions import APIException
 
 LOG = logging.getLogger('transmission')
 
 
-def engine_subscribe_generic(project, events=None):
+def engine_subscribe_generic(project, events=None, version=settings.LOAD_VERSION):
```

Comment:
The function name is called 'generic' but you are defaulting 'version' to be the LOAD contract version, which is not generic.

Reply:
The version parameter should come before `events` and have no default.